### PR TITLE
Add description for scrolling and scrolling.mode

### PR DESCRIPTION
**Pull request by pavel-2020**

### CODE REVIEW

File: api-reference/_hidden/dxSchedulerScrolling/mode.md [hunk 1]
```diff
@@ -6,7 +6,15 @@ default: 'standard'
...
-<!-- Description goes here -->
+Specifies the scrolling mode.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following scrolling modes are available:
+
+- **Standard**       
+All appointments are simultaneously loaded into the DOM and a whole grid is rendered.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/dxSchedulerScrolling/mode.md]:
```suggestion
All appointments are simultaneously loaded into the DOM, and the whole grid is rendered.
```

File: api-reference/_hidden/dxSchedulerScrolling/mode.md [hunk 2]
```diff
@@ -6,7 +6,15 @@ default: 'standard'
...
+
+- **Virtual**          
+Appointments are loaded into the DOM when they get into the viewport and removed once they leave it; a grid is rendered in the current viewport only.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/dxSchedulerScrolling/mode.md]:
```suggestion
Appointments are loaded into the DOM when they get into the viewport and removed once they leave it; the grid is rendered within the current viewport only.
```

File: api-reference/_hidden/dxSchedulerScrolling/mode.md [hunk 3]
```diff
@@ -6,7 +6,15 @@ default: 'standard'
...
+
+[note]The *"virtual"* mode is only available for the *"day"*, *"week"*, and *"workWeek"* [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/).
```
Comment (RomanTsukanov) [inline api-reference/_hidden/dxSchedulerScrolling/mode.md:20]: Do these views use the standard mode all the time?

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/scrolling.md [hunk 4]
```diff
@@ -5,7 +5,8 @@ inheritsType: dxSchedulerScrolling
...
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures scrolling.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Scrolling allows a user to browse data left outside the current viewport. The widget provides several scrolling modes detailed in the [mode](/Documentation/20_2/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/scrolling.md]:
```suggestion
Scrolling allows a user to browse data left outside the current viewport. The widget provides several scrolling modes detailed in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/scrolling.md [hunk 5]
```diff
@@ -5,7 +5,8 @@ inheritsType: dxSchedulerScrolling
...
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures scrolling.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Scrolling allows a user to browse data left outside the current viewport. The widget provides several scrolling modes detailed in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.
```
Comment (arminal) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/scrolling.md:12]: The widget contains several scrolling modes detailed in the [mode](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/#mode) option description.

File: api-reference/_hidden/dxSchedulerScrolling/mode.md [hunk 6]
```diff
@@ -6,7 +6,15 @@ default: 'standard'
...
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following scrolling modes are available:
+
+- **Standard**       
+All appointments are simultaneously loaded into the DOM, and the whole grid is rendered.
+
+- **Virtual**          
+Appointments are loaded into the DOM when they get into the viewport and removed once they leave it; the grid is rendered within the current viewport only.
```
Comment (arminal) [inline api-reference/_hidden/dxSchedulerScrolling/mode.md:18]: Appointments are loaded into the DOM when they get into the viewport and are then removed once they leave it; the grid is rendered within the current viewport only.

File: api-reference/_hidden/dxSchedulerScrolling/mode.md [hunk 7]
```diff
@@ -6,7 +6,15 @@ default: 'standard'
...
+
+[note]The *"virtual"* mode is only available for the *"day"*, *"week"*, and *"workWeek"* [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/).
```
Comment (arminal) [inline api-reference/_hidden/dxSchedulerScrolling/mode.md:20]: *"Virtual"* mode is only available for *"day"*, *"week"*, and *"workWeek"* [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/).

**Pull request merged**